### PR TITLE
feat(progressbar): add styles for progress label position

### PR DESCRIPTION
**Pull request by joneff**
Note: ~~currently, IE 11 is not so supported. I will add a follow up commit / branch to handle it.~~ Works in IE as well

There is no implicit position for the label. Instead, it relies on explicit label position.

/cc @Raisolution

### CODE REVIEW

File: tests/visual/progressbar.html [hunk 1]
```diff
@@ -161,22 +362,52 @@
                 </ul>
             </div>
 
-            <span></span>
-            <span>LTR</span>
-            <span>RTL</span>
-            <span>Indeterminate</span>
```
Comment (PreslavKozovski) [inline tests/visual/progressbar.html]: Why did you remove the Indeterminate progressbar?

Comment (joneff) [reply tests/visual/progressbar.html]: re-added
